core/finality-grandpa/src/environment.rs

```diff
@@ -462,7 +462,11 @@ where
 					// walk backwards until we find the target block
 					loop {
 						if *target_header.number() < target_number {
-							unreachable!();
+							unreachable!(
+								"we are traversing backwards from a know block; \
```

Comment:
typo: `know` → `known`

Reply:
Weird thing, I had the exact same message elsewhere but without the typo. Does this mean I did not copy-paste it when writing the code? :thinking::smile: